src/peft/peft_model.py

```diff
@@ -240,19 +240,24 @@ def save_pretrained(
                     f" {list(self.peft_config.keys())} - got {selected_adapters}."
                 )
 
-        def save_pissa_as_lora(peft_config, convert_pissa_to_lora, output_state_dict, kwargs):
-            if not str(peft_config.init_lora_weights).startswith("pissa"):
-                warnings.warn("`convert_pissa_to_lora` only works for converting a PiSSA adapter to a LoRA adapter")
-            initial_adapter = os.path.basename(convert_pissa_to_lora)
+        def save_mutated_as_lora(peft_config, convert_mutated_to_lora, output_state_dict, kwargs):
+            if not any(str(peft_config.init_lora_weights).startswith(prefix) for prefix in ["pissa", "olora"]):
```

Comment:
There is some inconsistency with regard to how the argument is called. Here it is assumed that it's `"olora"`, but in the example in the docs, it is `"OLoRA"`. How about making it case insensitive, so that both work? I.e. here we should call `.lower()` before comparing.

Reply:
That was my mistake. Everything should be lower case.